apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/service/PageHistoryFinder.java

```diff
@@ -0,0 +1,59 @@
+package gov.cdc.nbs.questionbank.page.service;
+
+import gov.cdc.nbs.questionbank.page.model.PageHistory;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class PageHistoryFinder {
+    private final JdbcTemplate jdbcTemplate;
+
+    private static final DateTimeFormatter dateFormatter = DateTimeFormatter
+            .ofPattern("MM/dd/yyyy")
+            .withZone(ZoneId.of("UTC"));
+
+    private static final String QUERY = """
+        SELECT
+        hist.publish_version_nbr AS publishVersionNbr,
+        hist.last_chg_time AS lastUpdatedDate,
+        userProfile.first_nm || ' ' || userProfile.last_nm AS lastUpdatedBy,
+        hist.version_note AS notes
+      FROM
+        WA_template_hist hist
+        LEFT OUTER JOIN user_profile userProfile ON hist.last_chg_user_id = userProfile.nedss_entry_id
+      WHERE
+        hist.wa_template_uid = ?
+      UNION
+      SELECT
+        tem.publish_version_nbr AS publishVersionNbr,
+        tem.last_chg_time AS lastUpdatedDate,
+        userProfile.first_nm || ' ' || userProfile.last_nm AS lastUpdatedBy,
+        tem.version_note AS notes
```

Comment:
Getting a syntax error for the `||`

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/c55a57d3-966e-47eb-afe3-0a03744ca893)